connector.go

```diff
@@ -202,6 +192,24 @@ func (wrb *WapiRequestBuilder) BuildBody(obj IBObject) []byte {
 		return nil
 	}
 
+	if t == GET && len(eaSearch) > 0 {
+		payload := make(map[string]string)
+		json.Unmarshal(jsonStr, &payload)
+		for k, v := range eaSearch {
+			str, ok := v.(string)
+			if !ok {
+				log.Printf("Cannot marshal EA Search attribute for '%s'\n", k)
+			} else {
+				payload["*"+k] = str
+			}
+		}
+		jsonStr, err = json.Marshal(payload)
+		if err != nil {
+			log.Printf("Cannot marshal EA's in the payload: '%s'", payload)
+			return nil
+		}
+	}
+
```

Comment:
Hi @yuewko , 
 I tried this but it will not work for our use case. 

Problem is that tag for the easearch will also come but we don't want that. We just want the k/v pairs of the easearch. 

For eg in `https://play.golang.org/p/XbC_RQLgO9` , we want the Marshalled json to be 
`{"Foo":"Bar","Name":"macvlan","CIDR":"10.10.0.0/24","Gateway":"10.10.0.1"}`
but the "EA" field is also coming. 
`{"EA":{"Foo":"Bar"},"Name":"macvlan","CIDR":"10.10.0.0/24","Gateway":"10.10.0.1"}`

This is currently not supported in the current encoding/json package. This functionality is proposed for the future release by addition of the 'inline' tag which does what we want.
`https://github.com/golang/go/issues/6213`